packages/gcf-utils/test/fixtures/repos.json

```diff
@@ -3,6 +3,7 @@
   "repos": [
     { "language": "cpp", "repo": "googleapis/cpp-cmakefiles"},
     { "language": "cpp", "repo": "googleapis/google-cloud-cpp"},
-    { "language": "cpp", "repo": "googleapis/google-cloud-cpp-bigquery"}
+    { "language": "cpp", "repo": "googleapis/google-cloud-cpp-bigquery"},
```

Comment:
added a fourth repo, so that we test the edge-case of the parallelization logic.